src/components/HomepageSandbox/index.js

```diff
@@ -15,6 +15,9 @@ export default function HomepageSandbox() {
         <Link to="https://www.cncf.io/projects/kubearmor/">
           <img className={styles.logo} src={sandboxImage} alt="logo" />
         </Link>
+        <h4 className={styles.sandboxText}>
```

Comment:
```suggestion
        <h4 className={styles.sandboxText} style={{ fontSize: "1rem" }}>
```

Reply:
Pls check and let me know.
Otherwise we will have to write some more styles to fix it.